src/utils/normalizeData.ts

```diff
@@ -1,20 +1,35 @@
-import type { OldListData, ListData, ListItem } from '../types/ListParams';
+import type { OldListData, ListData, ListItem, OldChecklistData } from '../types/ListParams';
 
 /**
- * Method that checks if data is related to the List or NestedListTool
- * @param data - data of the List or NestedListTool
- * @returns true if data related to the List tool, false if to Nested List tool
+ * Method that checks if data is related to the List Tool
+ * @param data - data of the List or NestedList tool
+ * @returns true if data related to the List tool, false otherwise
  */
-function instanceOfListData(data: ListData | OldListData): data is OldListData {
+function instanceOfListData(data: ListData | OldListData | OldChecklistData): data is OldListData {
```

Comment:
function name means checking passed value to be `ListData` (new). But it checks for the `OldListData`.